grumphp.yml

```diff
@@ -1,61 +0,0 @@
-parameters:
```

Comment:
Since we started to use a docker-compose setup everywhere, would it make sense to

```php
    git_hook_variables:
        EXEC_GRUMPHP_COMMAND: docker-compose run --rm application php
```

or would that be an application specific setting?